turbopack/crates/turbopack-tests/tests/snapshot/next-issues/74190/input/index.js

```diff
@@ -0,0 +1,15 @@
+var root = {};
```

Comment:
Could this be an execution test instead?

Reply:
I converted this into an execution test